src/deluge/model/instrument/midi_instrument.cpp

```diff
@@ -1237,19 +1237,25 @@ void MIDIInstrument::sendNoteToInternal(bool on, int32_t note, uint8_t velocity,
 	}
 }
 
-String* MIDIInstrument::getNameFromCC(int32_t cc) {
+std::string_view MIDIInstrument::getNameFromCC(int32_t cc) {
 	if (cc >= 0 && cc < kNumRealCCNumbers) {
-		MIDILabel* midiLabel = midiLabelCollection.labels.getLabelFromCC(cc);
```

Comment:
Looking at this on my phone and this change seems wrong here as if it's >=0 and < num real cc's then it's in range, not out of range

Will double check when I get in front of computer

Reply:
Fixed